vespalib/src/vespa/vespalib/stllike/string.h

```diff
@@ -431,6 +431,8 @@ class small_string
             _bufferSize = StackSize;
             if (s) {
                 memcpy(_stack, s, _sz);
+	    } else {
+                abort();
```

Comment:
Consider only aborting if _sz is nonzero. And add an abort in constructor at line 36 when `s` is null.

Reply:
It was intentionally rather strict, but I agree that it should be enough to mimic std::string.